chef/cookbooks/suse-manager-client/recipes/default.rb

```diff
@@ -36,11 +38,10 @@
   EOH
```

Comment:
Ideally yes. But I don't know if the rpm package is built the correct way for this.